css/templates/gallery/components/_sidebar.css

```diff
@@ -68,8 +70,20 @@
 }
 
 .gallery-sidebar .ampstart-navbar-trigger {
-  padding: calc(10 / 16rem);
-  margin: calc(-10 / 16rem);
+  height: var(--sidebar-nav-trigger-height);
+  line-height: calc(var(--sidebar-nav-trigger-height) * 1.066666667);
+  overflow: hidden;
+  padding: 0 .625rem;
+  margin-top: calc(
```

Comment:
I know this is more readable, but Google also won't like this. It needs to all be on one, giant, hard-to-read line (unfortunately)

Reply:
Ok cool. Even though it compiles to a single line?

Reply:
Yup, they're pretty strict about formatting